### ISSUE TITLE
Deck buffs are not counted for Twin Emperor's Battlecry

### ISSUE THREAD

**Issue by Kybxd** [HearthSim/hs-bugs#809]
* https://youtu.be/owj6qT95NoI

Another test: I have C'Thun in my deck and play 4 The Mistcallers. But Twin Emperor still cannot activate its battlecry.
I think C'Thun should apply a more intuitive mechanic, since The Mistcaller has been changed to suit its card text.

**Comment by troggnostupidhs** [comment]
I don't think this is a bug, it's how C'Thun works. If C'Thun is in the deck, the player (ritual) buffs are checked. If C'Thun is in your hand or on the battlefield, the cards attack is checked. 

C'Thun was in the deck at the time, so the Mistcaller buff isn't checked.

https://hearthstone.gamepedia.com/C%27Thun#Notes

> Cards with effects which activate based on C'Thun's stats, such as Ancient Shieldbearer and Klaxxi Amber-Weaver, will glow yellow and activate if at least one of the following are true:
> * Your 'ideal' (base + ritual buffs) C'Thun has 10+ Attack, even if you do not have a C'Thun (such as due to it being stolen, transformed, killed, or even never having been put into the deck to begin with).
> * You have a C'Thun in your hand or battlefield that is 10+ Attack, even using temporary buffs like Abusive Sergeant or The Mistcaller.

**Comment by troggnostupidhs** [comment]
https://us.battle.net/forums/en/hearthstone/topic/20762076199

> Keleseth's buff does not trigger C'thun's 10 attack cards (ie Klaxxi Amberweaver, Twin Emperor Vek'lor), even if C'thun has 10 attack. Quick use case scenario: I play Dark Arakkoa and Keleseth, but Amberweaver/Vek'lor do not trigger, even with C'thun at 10 attack.

---

> Issue is known, fix is being worked on.

**Comment by Jetz72** [comment]
Still happens as of 13.0.3:

![hearthstone screenshot 01-25-19 02 13 07](https://user-images.githubusercontent.com/8607832/51735608-db982100-2055-11e9-9760-6bf45a1b98f3.png)
![hearthstone screenshot 01-25-19 02 13 15](https://user-images.githubusercontent.com/8607832/51735614-de931180-2055-11e9-8cb6-e3605e1a161d.png)
![hearthstone screenshot 01-25-19 02 13 41](https://user-images.githubusercontent.com/8607832/51735642-ef438780-2055-11e9-9be3-77aaf453e884.png)
![hearthstone screenshot 01-25-19 02 15 06](https://user-images.githubusercontent.com/8607832/51735665-fa96b300-2055-11e9-89c8-f8a0d40c200a.png)

C'Thun had +3/+3 from disciples and +1/+1 from Mistcaller. Twin Emperor Vek'lor was not satisfied.

**Comment by Jetz72** [comment]
https://us.forums.blizzard.com/en/hearthstone/t/keleseth-not-working-with-cthun-buffs/3664
>The main points are:
I played Twilight Elder on turn 2 with coin: C’thun goes to 7/7
Played Skeram Cultist on 9: C’thun goes to 9/9
Played Keleseth on 10: C’thun still in deck, Keleseth triggered, C’thun should be at 10/10
Played Veklor on 11 even though he was lit up green, not yellow, and only got one Veklor
